apps/wpcom-block-editor/src/common/tracking/track-record-event.js

```diff
@@ -1,20 +1,30 @@
 /**
  * External dependencies
  */
-import { registerPlugin } from '@wordpress/plugins';
 import { isObjectLike, isUndefined, omit } from 'lodash';
 import debug from 'debug';
+
 const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
 
-// In case Tracks hasn't loaded
+// In case Tracks hasn't loaded.
 if ( typeof window !== 'undefined' ) {
 	window._tkq = window._tkq || [];
 }
 
 // Adapted from the analytics lib :(
 // Because this is happening outside of the Calypso app we can't reuse the same lib
 // This means we don't have any extra props like user
-const tracksRecordEvent = function( eventName, eventProperties ) {
+
+export default ( eventName, eventProperties ) => {
+	if ( undefined === window || undefined === window._currentSiteId ) {
+		return;
+	}
+
+	// Required by Tracks when added manually
+	// https://wpcom.trac.automattic.com/browser/trunk/wp-content/plugins/gutenberg-wpcom/index.php#L146
+	const blog_id = window._currentSiteId;
+	const site_type = window._currentSiteType;
```

Comment:
These values will only be available on wpcom multisite. In Jetpack sites this will be undefined.

Reply:
We can add misc values to the wpcomGutenberg object in Jetpack and wpcom but the new values won't be available until that Jetpack version is published, and folks upgrade to latest. https://github.com/Automattic/jetpack/blob/a388c6bef119c150b12f3900df6fe6d0e0d4eb43/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php#L274